### PR TITLE
Initialize project with components

### PR DESCRIPTION
**Pull request by Abenezer-Tilahun**
# Description

- In this milestone, I initialize the react bookstore app, update the read-me file, and set up the components needed for the project.

# General requirements

- There are no linter errors.
- Followed the GitFlow.
- The documentation is professional.

# Project requirements

- Followed best practices for JavaScript
- Project requirements
- Initialize the React app
- Add linter files and make sure it is working.
- Created a directory for the re-usable components
- Added React Router and set two Routes and Links for the app's navigation

### CODE REVIEW

File: src/component/BookList.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    return (
+      <ul id="book-list">
+        {
+          books.map((book) => (
+            <Book book={book} key={book} />
+          ))
+        }
+      </ul>
+    );
+  }
```
Comment (GzimAsani) [inline src/component/BookList.js]: - [x] This is throwing an error
Screenshot from 2022-03-31 21-52-00

- [x] You should create a `book`  component and render them in this component
- [x] You should also render here the `AddBookForm` component

File: src/component/BookList.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  return (
+    <>
+      There is no books to show.
```
Comment (GzimAsani) [inline src/component/BookList.js]: - [x] Remember when creating the book component:
- That it should take  the data as a prop and render the title and author
- Should have a button for removing the books(It is not necessary to work though)

File: src/component/AddBookForm.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            categories.forEach((cat) => (
+              <option value="volvo">{cat.title}</option>
+            ))
+          }
+        </select>
+        <button type="submit">Add book</button>
+      </form>
+    </section>
+  );
+}
```
Comment (GzimAsani) [inline src/component/AddBookForm.js]: - [x] This form should have two inputs for `title` and `author` of the book

File: src/component/Header.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Header.css';
+
+function Header() {
```
Comment (GzimAsani) [inline src/component/Header.js:5]: - [x] You should have a `Categories` component which should display only a button with text "Check status"

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi  @Abenezer-Tilahun                 
Good job so far! You are almost done :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good job with the git-flow :+1:
- Good PR description :clap: 

## Required Changes ♻️ 
- [x] _Check the comments below :arrow_double_down: _

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arslanbisharat** [comment]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: src/component/BookList.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/component/BookList.js]: I will do that . Thanks for suggestion.

File: src/component/BookList.js (hunk 2 above)
Comment (Abenezer-Tilahun) [reply src/component/BookList.js]: Done.

File: src/component/AddBookForm.js (hunk 3 above)
Comment (Abenezer-Tilahun) [reply src/component/AddBookForm.js]: Ok i will add input for title and author for book.

File: src/component/Header.js (hunk 4 above)
Comment (Abenezer-Tilahun) [reply src/component/Header.js:5]: I already include Categories component which should display only a button with text "Check status"

**Pull request merged**